libraries/GCS_MAVLink/MAVLink_routing.h

```diff
@@ -63,4 +63,6 @@ class MAVLink_routing
 
     // special handling for heartbeat messages
     void handle_heartbeat(mavlink_channel_t in_channel, const mavlink_message_t &msg);
+
+    void send_to_components(const char *pkt, const mavlink_msg_entry_t *entry);
```

Comment:
should we add a comment above this function?  the arguments are a little harder to understand than the previous one.  At least I'm not familiar with what a mavlink_msg_entry_t is.

Reply:
Commend added.